client/hostvolumemanager/host_volume_plugin.go

```diff
@@ -36,6 +38,10 @@ const (
 	EnvCapacityMin = "DHV_CAPACITY_MIN_BYTES"
 	EnvCapacityMax = "DHV_CAPACITY_MAX_BYTES"
 	EnvParameters  = "DHV_PARAMETERS"
+
+	// DefaultMkdirFileMode sets the mode of directories created by the
+	// "mkdir" built-in plugin to "0700"
+	DefaultMkdirFileMode os.FileMode = 0o700
```

Comment:
It seems this const is only used once within this file, so could we make it private?